test/e2e/kubectl/kubectl.go

```diff
@@ -471,7 +470,7 @@ var _ = SIGDescribe("Kubectl client", func() {
 
 			ginkgo.By("Starting http_proxy")
 			var proxyLogs bytes.Buffer
-			testSrv := httptest.NewServer(utilnettesting.NewHTTPProxyHandler(ginkgo.GinkgoT(), func(req *http.Request) bool {
+			testSrv := httptest.NewServer(NewHTTPProxyHandler(ginkgo.GinkgoT(), func(req *http.Request) bool {
```

Comment:
can we use `GinkgoTB()` instead of copying `NewHTTPProxyHandler`?

> `GinkgoTB() implements a wrapper that exactly matches the testing.TB interface.`

Reply:
duh, I was so shortsighted didn't realize is an interface

Reply:
ugh, I remember why, testing.TB implements an unexported method to avoid that but this got fixed with 

https://github.com/onsi/ginkgo/issues/582 TIL